IM150/admin/admin_forum_tour.php

```diff
@@ -163,7 +165,7 @@ function reorder_pages()
 			$message = prepare_message(trim($message), 1, 1, 1, $bbcode_uid);
 
 			$sql = "UPDATE " . FORUM_TOUR_TABLE . "
-				SET page_subject = '$subject', page_text = '$message', page_access = $page_access
+				SET page_subject = '$subject', page_text = '$message', page_access = '$page_access'

```

Comment:
Page access is anonymous, user, mod, admin . Not sure how the different user levels matter other than if you are not logged in you get a basic go away message. The page_access is storing in the database as an integer.